### ISSUE TITLE
Review use of application headers in 'No data' views

### ISSUE THREAD

**Issue by rkzel** [AutarkLabs/open-enterprise#1850]
A11Y moderate issue: Page must contain a level-one heading

Description (which explains what's missing and why it is useful to be present):

> Screen reader users can use keyboard shortcuts to navigate directly to the first h1, which, in principle, should allow them to jump directly to the main content of the web page. If there is no h1, or if the h1 appears somewhere other than at the start of the main content, screen reader users must listen to more of the web page to understand its structure, wasting valuable time.

All of our app display default "no data" card when there is nothing to display (when the app is freshly installed, etc):

![Screenshot from 2020-01-08 14-33-28.png](https://images.zenhubusercontent.com/5ad4f5d94b5806bc2bcf53cf/7e92fc35-7069-4a9e-a766-c6cd74810cf4)

We can add a defaul Header, Info box, or a hidden div in "header" role.

**Comment by javieralaves** [comment]
@stellarmagnet I don't have any strong objections for placing the app name as the header (without the global button on the right side for as long as there's an empty card right below it with the same button). What do you think?
